junit5/src/main/java/com/linecorp/armeria/testing/server/ServiceRequestContextCaptor.java

```diff
@@ -172,4 +172,11 @@ public ServiceRequestContext poll() throws InterruptedException {
     public ServiceRequestContext poll(long timeout, TimeUnit unit) throws InterruptedException {
         return serviceContexts.poll(timeout, unit);
     }
+
+    /**
+     * Returns whether the {@link #size()} of this data is 0.
+     */
+    public boolean isEmpty() {
+        return size() == 0;
+    }
```

Comment:
We should use `isEmpty()` instead of `size() == 0` because it's potentially more efficient.

```suggestion
    /**
     * Returns whether there is any captured {@link ServiceRequestContext}.
     */
    public boolean isEmpty() {
        return serviceContexts.isEmpty();
    }
```